.github/workflows/coding.yml

```
@@ -2,7 +2,7 @@
 
 on:
   push:
-    branches: [ "coding*" ]
+    branches: [ "coding*", "prog*", "test*" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)